src/components/register.js

```diff
@@ -56,9 +56,90 @@ return (
                 }}
         placeholder={"Enter password"}/>
 
-        <button>Register</button>
+        {/* <button>Register</button> */}
```

Comment:
Just like login, this button submits the form so we will need it or some other way to do so

Reply:
Good catch and thanks for pointing that out.